util/shortrfc3339/shortrfc3339.go

```diff
@@ -0,0 +1,36 @@
+// Package shortrfc3339 implements helpers for working with shortened RFC-3339 compliant timestamps (those without seconds).
+package shortrfc3339
+
+import (
+	"strings"
+	"time"
+)
+
+// ShortRFC3339Timestamp is a custom JSON encoder / decoder for shortened RFC3339-compliant timestamps (those without seconds).
+type ShortRFC3339Timestamp struct {
```

Comment:
Still wondering- but didn't try- if we could make this 

    type ShortRFC3339Timestamp time.Time

which might allow us to reuse `IsZero`?

Reply:
Uhhhh, I don't _think_ so judging by the little bit of IDE juggling I just did, doesn't appear to properly extend - feel free to correct me on that though

Also `ShortRFC3339Timestamp.IsZero()` is still an available function, see `ShortRFC3339Timestamp.IsSet()`

Reply:
Concur, d987910fae424681ead4f37d120b386634f3b6f